ingestion_server/ingestion_server/indexer.py

```diff
@@ -488,6 +488,21 @@ def create_and_populate_filtered_index(
         destination_index_suffix: str | None = None,
         **_,
     ):
+        """
+        Create and populate a filtered index without documents with sensitive terms.
+
+        :param model_name: The model/media type the filtered index is for.
+        :param origin_index_suffix: The suffix of the origin index on which the
+        filtered index should be based. If not supplied, the filtered index will be
+        based upon the index aliased to ``model_name`` at the time of execution.
+        :param destination_index_suffix: The suffix to use for the destination index.
+        This can be configured independently of the origin index suffix to allow for
+        multiple filtered index creations to occur against the same origin index, i.e.,
+        in the case where sensitive terms change before a data refresh creates a new
+        origin index and we wish to update the filtered index immediately. If not
+        supplied, a UUID based suffix will be generated. This does not affect the
+        final alias used.
+        """
```

Comment:
I've added this documentation so that the Airflow param docs can link to the code to indicate how they are used upstream rather than needing to repeat that information.